Controller/Adminhtml/Exportcategory/Export.php

```diff
@@ -90,7 +92,7 @@ public function execute()
                     }
                     $content .= '"'.$categoryitem->getId().'","'.$categoryitem->getParentId().'","';
                     $content .= $_stores[$categoryitem->getStoreId()].'","';
-                    $content .= $categoryitem->getName().'","'.$categoryitem->getPath().'","'.$categoryitem->getImage().'","'.$categoryitem->getIsActive().'","'.$categoryitem->getIsAnchor().'","'.$categoryitem->getIncludeInMenu().'","'.$categoryitem->getMetaTitle().'","'.$categoryitem->getMetaKeywords().'","'.$categoryitem->getMetaDescription().'","'.$categoryitem->getDisplayMode().'","'.$categoryitem->getCustomUseParentSettings().'","'.$categoryitem->getCustomApplyToProducts().'","'.$categoryitem->getCustomDesign().'","'.$categoryitem->getCustomDesignFrom().'","'.$categoryitem->getCustomDesignTo().'","'.$categoryitem->getDefaultSortBy().'","'.$categoryitem->getPageLayout().'","'.$categoryitem->getDescription().'","'.$prodids.'"'."\n";
+                    $content .= $categoryitem->getName().'","'.$categoryitem->getPath().'","'.$categoryitem->getImage().'","'.$categoryitem->getUrlKey().'",'.$categoryitem->getIsActive().'","'.$categoryitem->getIsAnchor().'","'.$categoryitem->getIncludeInMenu().'","'.$categoryitem->getMetaTitle().'","'.$categoryitem->getMetaKeywords().'","'.$categoryitem->getMetaDescription().'","'.$categoryitem->getDisplayMode().'","'.$categoryitem->getCustomUseParentSettings().'","'.$categoryitem->getCustomApplyToProducts().'","'.$categoryitem->getCustomDesign().'","'.$categoryitem->getCustomDesignFrom().'","'.$categoryitem->getCustomDesignTo().'","'.$categoryitem->getDefaultSortBy().'","'.$categoryitem->getPageLayout().'","'.$categoryitem->getDescription().'","'.$prodids.'"'."\n";
```

Comment:
Please stick to the 120 character limit.

Reply:
addressed i next commit